### PR TITLE
doc: Add description for the RGB-LED and the buttons.

### PR DESCRIPTION
**Pull request by RomanBuchert**

### CODE REVIEW

File: doc/source/design/board/gd32vf103c_longan_nano.rst [hunk 1]
```diff
@@ -20,6 +20,22 @@ Is is similar to the well known STM32-based "Blue Pill" board.
...
+.. _design_board_sipeed_longan_nano_versions:
+
+Versions
+~~~~~~~~
+
+There are two versions of this board available.
+
+* GD32VF103C\ **B**\ T6 with 128k Flash / 32k RAM
+
+* GD32VF103C\ **8**\ T6 with 64k Flash / 20k RAM. This is sometimes called the "lite" version.
```
Comment (fanghuaqi) [inline doc/source/design/board/gd32vf103c_longan_nano.rst:33]: The PR #18 will introduce the `VARIANT` variable, so you can add some note here to let user know how to select the version developer want to use.

File: doc/source/design/board/gd32vf103c_longan_nano.rst [hunk 2]
```diff
@@ -79,7 +95,7 @@ are fixed to ``flashxip`` and ``n205``.
 * There are two versions of this board avail. One with 64k flash and one with 128k flash.
 
   - You have to change the linker script to set the right flash size. By default, 128k flash is set.
-  
+
 * You can find its linker script in ``SoC/gd32vf103/Board/gd32vf103c_longan_nano/Source/GCC/``
 
   - ``gcc_gd32vf103_flashxip.ld``: Linker script file for ``DOWNLOAD=flashxip``
```
Comment (fanghuaqi) [inline doc/source/design/board/gd32vf103c_longan_nano.rst:101]: The linker script file are different for different version for PR #18 , could you also add some notes here.

**Review by fanghuaqi — CHANGES_REQUESTED** [review]
Help to update the documentation when PR #18 is merged. @RomanBuchert @tuupola

File: doc/source/design/board/gd32vf103c_longan_nano.rst (hunk 2 above)
Comment (RomanBuchert) [reply doc/source/design/board/gd32vf103c_longan_nano.rst:101]: I think if you can merge this PR, than @tuupola can add the description of his work. Would this be ok?

Comment (fanghuaqi) [reply doc/source/design/board/gd32vf103c_longan_nano.rst:101]: That is ok for me.  I will merge it now.

**Pull request merged**